wiki/en/misc/general.yml

```diff
@@ -1,5 +1,5 @@
 footer:
-  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>
+  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a> - want to <a href="/wiki/Contribution">get involved?</a> <div id="asio" style="margin-top:20px;">ASIO is a trademark and software of Steinberg Media Technologies GmbH <img alt="ASIO logo" src="/assets/img/asio.jpg" style="display:block;margin-left:auto;margin-right:auto;width:70px;height:auto;" loading="lazy"></div>
```

Comment:
I'd add it to the footer in the footer inclusion file: https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/_includes/footer.html

Reply:
Sorry, what do you mean? Do you mean footer.html is a better place for it than general.yml?

Reply:
Yes